src/Jackett.Common/Definitions/mma-torrents.yml

```diff
@@ -0,0 +1,164 @@
+      - {id: 131, cat: TV/Sport, desc: "Events - Cage Warriors"}
+      - {id: 132, cat: TV/Sport, desc: "Events - Invicta FC"}
+      - {id: 130, cat: TV/Sport, desc: "Events - KOTC"}
+      - {id: 90, cat: TV/Sport, desc: "Events - M-1"}
+      - {id: 120, cat: TV/Sport, desc: "Events - MMA/Brazil"}
+      - {id: 70, cat: TV/Sport, desc: "Events - MMA/Other/HD"}
+      - {id: 47, cat: TV/Sport, desc: "Events - MMA/Other/SD"}
+      - {id: 129, cat: TV/Sport, desc: "Events - ONE"}
+      - {id: 126, cat: TV/Sport, desc: "Events - Other/Other"}
+      - {id: 96, cat: TV/Sport, desc: "Events - UFC/HD"}
+      - {id: 103, cat: TV/Sport, desc: "Events - UFC/SD"}
+      - {id: 93, cat: Audio/Other, desc: "Instructional - Audio"}
+      - {id: 112, cat: TV/Sport, desc: "Instructional - DVD/ISO/WEB"}
+      - {id: 113, cat: Movies/DVD, desc: "Instructional - DVDRip/Other"}
+      - {id: 73, cat: Books/Ebook, desc: "Instructional - E-book"}
+      - {id: 83, cat: TV/OTHER, desc: "Instructional - Fitness"}
+      - {id: 124, cat: TV/Sport, desc: "JJ/BJJ - Events"}
+      - {id: 85, cat: TV/Sport, desc: "JJ/BJJ - Misc/Other"}
+      - {id: 99, cat: TV/Sport, desc: "Kickboxing - HD"}
+      - {id: 84, cat: TV/Sport, desc: "Kickboxing - SD"}
+      - {id: 111, cat: Books/Magazines, desc: "Magazines - All"}
+      - {id: 58, cat: Other/Misc, desc: "Misc - All"}
+      - {id: 63, cat: TV/Sport, desc: "Misc - MMA Special"}
+      - {id: 62, cat: Movies/Other, desc: "Movies - MMA Related"}
+      - {id: 77, cat: TV/Sport, desc: "Muay Thai - All"}
+      - {id: 110, cat: TV/Sport, desc: "Packs - Career"}
+      - {id: 115, cat: TV/Sport, desc: "Packs - Instructionals"}
+      - {id: 91, cat: TV/Sport, desc: "Packs - Other/HD"}
+      - {id: 64, cat: TV/Sport, desc: "Packs - Other/SD"}
+      - {id: 95, cat: TV/Sport, desc: "Packs - Pride"}
+      - {id: 59, cat: TV/Sport, desc: "Packs - TUF"}
+      - {id: 114, cat: TV/Sport, desc: "Packs - UFC"}
+      - {id: 122, cat: TV/Sport, desc: "Packs - Userpacks"}
+      - {id: 72, cat: TV/Sport, desc: "Retro - K-1"}
+      - {id: 61, cat: TV/Sport, desc: "Retro - Other/All"}
+      - {id: 66, cat: TV/Sport, desc: "Retro - PRIDE"}
+      - {id: 67, cat: TV/Sport, desc: "Retro - UFC"}
+      - {id: 69, cat: TV/Sport, desc: "Retro - WEC"}
+      - {id: 97, cat: TV/Sport, desc: "TUF - Episode/HD"}
+      - {id: 104, cat: TV/Sport, desc: "TUF - Episode/SD"}
+      - {id: 101, cat: TV/Sport, desc: "TV Show - Episode/HD"}
+      - {id: 94, cat: TV/Sport, desc: "TV Show - Episode/SD"}
+      - {id: 127, cat: TV/Sport, desc: "TV Show - WebRip"}
+
+    modes:
+      search: [q]
+      tv-search: [q, season, ep]
+
+  settings:
+    - name: username
+      type: text
+      label: Username
+    - name: password
+      type: password
+      label: Password
+    - name: sort
+      type: select
+      label: Sort requested from site
+      default: "seed"
+      options:
```

Comment:
is the torrents.php not capable of sorting by torrent id/added/uploaded date stamps?
the default is usually the option that corresponds to "created" since this is the default order that the jackett dashboard presents the results on.

Reply:
@garfield69 the "created" field is where I had to get a bit goofy, but the dateparse was mostly functioning.  I did get some errors, and thus the "optional" field while working through things.

Instead of posting as a default TR/Column, there are a few fields which are embedded as text within the second Column which represents the title.

`Size: 836.77 MB
Speed: No Activity
Added: 2020-01-24 20:35:13
Completed: 6`

Reply:
Unable to upload the txt file without it getting annoyed at me

`<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">



<HTML xmlns="http://www.w3.org/1999/xhtml">



<head>



<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">



<title>MMA-Torrents.com - Mixed Martial Arts Tracker  : Browse Torrents</title>



<meta name="description" content="As Real As It Gets - The Best MMA Torrents Around. We are what MMA is all about. Latest News, Discuss Fighters, Discuss Fights, MMA-Torrents.com Taking you to the fights, this is the one you dont want to miss..." />

<meta name="keywords" content="Best MMA Torrents, MMA Torrent, MMA Torrents, MMA, UFC, Ultimate Fighting Championship, MMA Streams, MMA News, MMA-Torrents.com, Ultimate MMA, The Best of MMA, Greatest MMA, TOP MMA, Mixed Martial Arts torrents, UFC Torrents, Bellator, Fight Night, Boxing, Classic MMA, WEC and more... " />

<meta name="googlebot" content="all, index, follow"></meta>

<meta name="pagerank" content="10"></meta>

<script type="text/javascript" src="https://mma-torrents.com/backend/java_klappe.js"></script>


<link rel="stylesheet" type="text/css" href="https://mma-torrents.com/themes/NB-MMA/theme.css">



<link rel='alternate stylesheet' type='text/css' media='screen' title='orange' href='themes/NB-MMA/orange.css' />



<link rel='alternate stylesheet' type='text/css' media='screen' title='red' href='themes/NB-MMA/red.css' />



<link rel='alternate stylesheet' type='text/css' media='screen' title='green' href='themes/NB-MMA/green.css' />



<link rel='alternate stylesheet' type='text/css' media='screen' title='blue' href='themes/NB-MMA/blue.css' />



<script src='themes/NB-MMA/js/styleswitch.js' type='text/javascript'></script>



    <script type='text/javascript' src='themes/NB-MMA/js/jquery-1.1.3.1.min.js'></script>



    <script type='text/javascript' src='themes/NB-MMA/js/jquery.easing.min.js'></script>



    <script type='text/javascript' src='themes/NB-MMA/js/jquery.lavalamp.min.js'></script>



    <script type='text/javascript'>



        $(function() {



            $('#lnav').lavaLamp({



                fx: 'backout',



                speed: 700,



                click: function(event, menuItem) {



                    return true;



                }



            });



        });



    </script>



</head>







<BODY>



<a name="top"></a>



<div id='wrapper'>



<table width="80%" align="center" border="0" cellspacing="0" cellpadding="0">



  <tr>



    <td height="1" bgcolor="#000000"><img src="themes/NB-MMA/images/blank.gif" width="1200" height="1" /></td>



  </tr>



  <tr>



    <td valign="top">



  <div id='container'>



    <div id='header'>



        <div class='col-nav'>



           <form action='#' method='post'>



             <input type='button' value='' title="Orange" name='choice' id='orange' onclick='chooseStyle(this.id, 60)'></input>



             <input type='button' value='' title="Red" name='choice' id='red' onclick='chooseStyle(this.id, 60)'></input>



             <input type='button' value='' title="Green" name='choice' id='green' onclick='chooseStyle(this.id, 60)'></input>



             <input type='button' value='' title="Blue" name='choice' id='blue' onclick='chooseStyle(this.id, 60)'></input>



           </form>



      </div>



    </div>



<table width='100%' border='0' cellspacing='0' cellpadding='0'>



  <tr>



    <td width='60' height='54'><img src='themes/NB-MMA/images/nav-l.png' width='60' height='54' /></td>



    <td height='54' align='center' valign='top' background='themes/NB-MMA/images/nav-bg.png'><table border='0' cellspacing='0' cellpadding='0'>



      <tr>



        <td height='54'>



        <ul class='lavaLamp' id='lnav' name='lnav'>



        <li><a href='/' target='_top'><span>Home</span></a></li>



        <li><a href='torrents.php' target='_top'><span>Browse</span></a></li>



        <li><a href='foru' target='_top'><span>Forum</span></a></li>



        <li><a href='faq.php' target='_top'><span>FAQ</span></a></li>



        <li><a href='rules.php' target='_top'><span>Rules</span></a></li>



        <li><a href='http://mma-portal.com' target='_blank'><span>News</span></a></li>



        <li><a href='torrents-today.php' target='_top'><span>Today</span></a></li>



        <li><a href='chat.php' target='_top'><span>Webchat</span></a></li>



        <li><a href='webirc.php' target='_top'><span>Irchat</span></a></li>



        <li><a href='torrents-upload.php' target='_top'><span>Upload</span></a></li>


        </ul>
        </td>



      </tr>



    </table></td>



    <td width='60' height='54'><img src='themes/NB-MMA/images/nav-r.png' width='60' height='54' /></td>



  </tr>



</table>



<table width='100%' border='0' cellspacing='0' cellpadding='0'>



  <tr>



    <td width='10' height='32'><img src='themes/NB-MMA/images/main-tl.png' width='10' height='32' /></td>



    <td height='32' background='themes/NB-MMA/images/main-tm.jpg'>



    <table id="infobar" border='0' align="left" cellpadding='0' cellspacing='0'>

  <tr>

    <td width='8' height='32'><img src='themes/NB-MMA/images/info-l.gif' width='8' height='32' /></td>

    <td id="info" height='32' background='themes/NB-MMA/images/info-m.gif'>

<!-- START INFOBAR CODE -->



You are logged in as: ---PII-USERNAME---&nbsp; <a href='account.php'><img src='/img/account.gif' border='none' align='absmiddle' height='16' width='16' alt='Your account' title='Your account'></a> <a href='account-details.php?id=23323'><img src='/img/profile.gif' border='none' align='absmiddle' height='16' width='16' alt='Profile' title='Profile'></a> <a href="account-logout.php"><img src='/img/greylogout.gif' border='none' align='absmiddle' height='16' width='16' alt='Logout' title='Logout'></a> <font color=#FF0000><B><img src='/img/greynewpm.gif' border='none' align='absmiddle' height='16' width='16' alt='New PM' title='New PM'> (<a href=mailbox.php?inbox>4</a>)</b></a></font>&nbsp;

<!-- END INFOBAR CODE -->

    </td>

    <td width='8' height='32'><img src='themes/NB-MMA/images/info-r.gif' width='8' height='32' /></td>

  </tr>

</table>



	






    <div class='mySearch'>



    <form method="get" action="torrents.php">



    <fieldset>



    <input type='text'  value="Search ..." onfocus="if (this.value == 'Search ...') this.value = '';" onblur="if (this.value == '') this.value = 'Search ...';" name='search' id='q' class='mySearch-t' />



	<input name="incldead" value="1" type="hidden">



    <input type='submit' value='Submit' class='mySearch-b' />



    </fieldset>



    </form>



    </div>



    


    </td>



    <td width='10' height='32'><img src='themes/NB-MMA/images/main-tr.png' /></td>



  </tr>



</table>



<table width='100%' border='0' cellspacing='0' cellpadding='0'>



  <tr>



    <td width='4' background='themes/NB-MMA/images/main-l.png'>&nbsp;</td>



    <td valign='top' bgcolor='#1b1d20'><!-- START CONTENT CODE -->



<TABLE cellSpacing="6" cellPadding="0" width="100%" border="0" >



	<TBODY>



	<TR>







	


	<TD id="lcol" vAlign="top" width="180">



	<div class='blok'>
<div class='blok-t'>---PII-USERNAME---</div>
<div class='blok-m'>
<div class='blok-content'><center><img width=90 height=90 src=https://mma-torrents.com/images/default_avatar.gif><br>Your Ratio: --- </center></div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>Navigation</div>
<div class='blok-m'>
<div class='blok-content'>- <a href=index.php>Home</a><BR>- <a href=torrents.php>Browse Torrents</a><BR>- <a href=torrents-today.php>Todays Torrents</a><BR>- <a href=torrents-needseed.php>Needing Seeds</a><BR>- <a href=foru/viewforum.php?f=83 target=_blank>Guides & Tutorials</a><BR>-<BR>- <a href=staff.php>Staff</a><BR>- <a href=teams-view.php>Our Teams</a><BR>- <a href=avatarwall.php>Avatar Wall</a><BR>- <a href=invite.php>Invites</a><BR>- <a href=rules.php>Rules</a><BR>- <a href=faq.php>FAQ</a><BR></div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>MMA Quotes</div>
<div class='blok-m'>
<div class='blok-content'><center>Boxing is a road map of what not to do. The greedy promoters basically killed the sport by taking it off free TV. - Dana White
</center></div>
<div class='blok-bottom'></div>
</div>
</div>
<div class='blok'>
<div class='blok-t'>Seeders Wanted</div>
<div class='blok-m'>
<div class='blok-content'><a href='torrents-details.php?id=50371'>Rick tucci kali dv...</A><BR> - Leechers Waitng: [9]<BR>
<a href='torrents-details.php?id=49967'>De La Riva and Rev...</A><BR> - Leechers Waitng: [5]<BR>
<a href='torrents-details.php?id=17499'>Roger Gracie - Com...</A><BR> - Leechers Waitng: [4]<BR>
<a href='torrents-details.php?id=9697'>Eddie Bravo - Mast...</A><BR> - Leechers Waitng: [3]<BR>
<a href='torrents-details.php?id=19011'>Marcelo Garcia - C...</A><BR> - Leechers Waitng: [3]<BR>
</div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>Latest Posters</div>
<div class='blok-m'>
<div class='blok-content'><table align=center cellpadding=0 cellspacing=0 width=100% border=0><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50703'><img border='0' src='uploads/images/507030.jpg' title="Creating Kuzushi by Israel Hernandez / DVD/ISO/WEB" alt="Creating Kuzushi by Israel Hernandez / DVD/ISO/WEB" width=100></a><BR></td></tr><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50704'><img border='0' src='uploads/images/507040.jpg' title="Wing Chun Gary Lam / DVDRip/Other" alt="Wing Chun Gary Lam / DVDRip/Other" width=100></a><BR></td></tr><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50705'><img border='0' src='uploads/images/507050.jpg' title="Warfare.MMA.18.720p.WEB.DL.x264-ThS / MMA/Other/HD" alt="Warfare.MMA.18.720p.WEB.DL.x264-ThS / MMA/Other/HD" width=100></a><BR></td></tr><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50706'><img border='0' src='uploads/images/507060.jpg' title="UFC.248.Embedded-Vlog.Series-Episode.5.720p.WEBRip.h264-TJ / UFC/HD" alt="UFC.248.Embedded-Vlog.Series-Episode.5.720p.WEBRip.h264-TJ / UFC/HD" width=100></a><BR></td></tr><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50724'><img border='0' src='uploads/images/507240.jpg' title="We.Love.MMA.52.720p.WEB.DL.x264-ThS / MMA/Other/HD" alt="We.Love.MMA.52.720p.WEB.DL.x264-ThS / MMA/Other/HD" width=100></a><BR></td></tr><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50725'><img border='0' src='uploads/images/507250.jpg' title="UFC.248.Embedded-Vlog.Series-Episode.6.1080p.WEBRip.h264-TJ / UFC/HD" alt="UFC.248.Embedded-Vlog.Series-Episode.6.1080p.WEBRip.h264-TJ / UFC/HD" width=100></a><BR></td></tr><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50726'><img border='0' src='uploads/images/507260.jpg' title="NFC.123.720p.WEB.DL.x264-ThS / MMA/Other/HD" alt="NFC.123.720p.WEB.DL.x264-ThS / MMA/Other/HD" width=100></a><BR></td></tr><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50728'><img border='0' src='uploads/images/507280.jpg' title="ACA.105.720p.WEB.DL.x264-ThS / MMA/Other/HD" alt="ACA.105.720p.WEB.DL.x264-ThS / MMA/Other/HD" width=100></a><BR></td></tr><tr><td align=center><a href='https://mma-torrents.com/torrents-details.php?id=50751'><img border='0' src='uploads/images/507510.jpg' title="UFC 248 Prelims 1080p HDTV DD 2.0 H.264-JustHD / UFC/HD" alt="UFC 248 Prelims 1080p HDTV DD 2.0 H.264-JustHD / UFC/HD" width=100></a><BR></td></tr></table></div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>Invites</div>
<div class='blok-m'>
<div class='blok-content'>	<table border="0" width="100%" cellspacing="0" cellpadding="0">
	<tr><td align="center">You Have
 1 Invites
<br></td></tr>
		<tr><td align="center"><a href=invite.php>Send an invite
</a><br></td></tr>
		</table>
	</div>
<div class='blok-bottom'></div>
</div>
</div>


	</TD>



	






	<TD id="mcol" vAlign="top"><!-- MAIN CENTER CONTENT START -->







	
     
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-20272611-3"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-20272611-3');
</script>




<div class='fram'>
<div>
<div class='fram-tl'></div>
<div class='fram-tr'></div>
<div class='fram-tm'>
<div class='fram-caption'>Browse Torrents</div>
</div>
</div>
<div>
<table width='100%' border='0' cellspacing='0' cellpadding='0'>
<tr>
<td width='4' valign='top' background='themes/NB-MMA/images/fml.gif'><img src='themes/NB-MMA/images/fmlt.gif' width='4' height='12' /></td>
<td class='fram-m' valign='top'>
<div class='fram-content'><center><p>Categories Open/Close</p></center>
<form method="get" action="torrents.php">
<center>&nbsp;<img src='images/plus.gif' id='pic1' onclick='klappe_torrent(1)'></center><div id='k1' style='display: none;'><table class=bottom align="center">
<tr align='right'>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=92><span class=newlink>Audio - Podcast</span></a><input name=c92 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=128><span class=newlink>Boxing - Bare Knuckle</span></a><input name=c128 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=105><span class=newlink>Boxing - HD</span></a><input name=c105 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=71><span class=newlink>Boxing - SD</span></a><input name=c71 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=86><span class=newlink>Documentary - All</span></a><input name=c86 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=109><span class=newlink>Events - Bellator/HD</span></a><input name=c109 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=89><span class=newlink>Events - Bellator/SD</span></a><input name=c89 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=131><span class=newlink>Events - Cage Warriors</span></a><input name=c131 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=132><span class=newlink>Events - Invicta FC</span></a><input name=c132 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=130><span class=newlink>Events - KOTC</span></a><input name=c130 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=90><span class=newlink>Events - M-1</span></a><input name=c90 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=120><span class=newlink>Events - MMA/Brazil</span></a><input name=c120 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=70><span class=newlink>Events - MMA/Other/HD</span></a><input name=c70 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=47><span class=newlink>Events - MMA/Other/SD</span></a><input name=c47 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=129><span class=newlink>Events - ONE</span></a><input name=c129 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=126><span class=newlink>Events - Other/Other</span></a><input name=c126 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=96><span class=newlink>Events - UFC/HD</span></a><input name=c96 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=103><span class=newlink>Events - UFC/SD</span></a><input name=c103 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=93><span class=newlink>Instructional - Audio</span></a><input name=c93 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=112><span class=newlink>Instructional - DVD/ISO/WEB</span></a><input name=c112 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=113><span class=newlink>Instructional - DVDRip/Other</span></a><input name=c113 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=73><span class=newlink>Instructional - E-book</span></a><input name=c73 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=83><span class=newlink>Instructional - Fitness</span></a><input name=c83 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=124><span class=newlink>JJ/BJJ - Events</span></a><input name=c124 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=85><span class=newlink>JJ/BJJ - Misc/Other</span></a><input name=c85 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=99><span class=newlink>Kickboxing - HD</span></a><input name=c99 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=84><span class=newlink>Kickboxing - SD</span></a><input name=c84 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=111><span class=newlink>Magazines - All</span></a><input name=c111 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=58><span class=newlink>Misc - All</span></a><input name=c58 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=63><span class=newlink>Misc - MMA Special</span></a><input name=c63 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=62><span class=newlink>Movies - MMA Related</span></a><input name=c62 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=77><span class=newlink>Muay Thai - All</span></a><input name=c77 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=110><span class=newlink>Packs - Career</span></a><input name=c110 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=115><span class=newlink>Packs - Instructionals</span></a><input name=c115 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=91><span class=newlink>Packs - Other/HD</span></a><input name=c91 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=64><span class=newlink>Packs - Other/SD</span></a><input name=c64 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=95><span class=newlink>Packs - Pride</span></a><input name=c95 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=59><span class=newlink>Packs - TUF</span></a><input name=c59 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=114><span class=newlink>Packs - UFC</span></a><input name=c114 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=122><span class=newlink>Packs - Userpacks</span></a><input name=c122 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=72><span class=newlink>Retro - K-1</span></a><input name=c72 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=61><span class=newlink>Retro - Other/All</span></a><input name=c61 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=66><span class=newlink>Retro - PRIDE</span></a><input name=c66 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=67><span class=newlink>Retro - UFC</span></a><input name=c67 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=69><span class=newlink>Retro - WEC</span></a><input name=c69 type="checkbox" value=1></td>
</tr><tr align='right'><td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=97><span class=newlink>TUF - Episode/HD</span></a><input name=c97 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=104><span class=newlink>TUF - Episode/SD</span></a><input name=c104 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=101><span class=newlink>TV Show - Episode/HD</span></a><input name=c101 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=94><span class=newlink>TV Show - Episode/SD</span></a><input name=c94 type="checkbox" value=1></td>
<td style="padding-bottom: 2px;padding-left: 2px"><a class=catlink href=torrents.php?incldead=1&cat=127><span class=newlink>TV Show - WebRip</span></a><input name=c127 type="checkbox" value=1></td>
</tr></table></CENTER><BR>	<CENTER>
	Search
	<input type="text" name="search" size="40" value="ufc 245" />

    <input type="hidden" name="incldead" value="1">
    <input type="hidden" name="lang" value="0">

    <select name="freeleech">
    <option value="0">All</option>
    <option value="1" >Not freeleech</option>
    <option value="2" >Only freeleech</option>
     </select>
	<input type="submit" value="GO
" />
	<br>
	</form>
	<br/>
	</div>

<p align="center"><span class=newlink>1&nbsp;-&nbsp;50</span> | <a href="torrents.php?incldead=1&amp;search=ufc+245&amp;sort=id&order=desc&page=1"><span class=newlink>51&nbsp;-&nbsp;59</span></a><br /><br><span class=newlink>&lt;&lt;&nbsp;Prev</span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="torrents.php?incldead=1&amp;search=ufc+245&amp;sort=id&order=desc&page=1"><span class=newlink>Next&nbsp;&gt;&gt;</span><br></a></p>
<br>
	<table align=center cellpadding="0" cellspacing="0" class="ttable_headinner" width=99%>

	<td class=ttable_head>Type</td>

	<td class=ttable_head><a href="?search=ufc 245&incldead=1&sort=name&order=desc" title="Name">Name</a></td>

	<td class=ttable_head>DL</td>

	<td class=ttable_head>NFO</td>

    <td class=ttable_head><a href="?search=ufc 245&incldead=1&sort=comments&order=desc" title="Comments">C</a></td>

    <td class=ttable_head><a href="?search=ufc 245&incldead=1&sort=size&order=desc" title="Size">Size</a></td>

	<td class=ttable_head><a href="?search=ufc 245&incldead=1&sort=seeders&order=desc" title="Seeders">S</a></td>

	<td class=ttable_head><a href="?search=ufc 245&incldead=1&sort=leechers&order=desc" title="Leechers">L</a></td>

	<td class=ttable_head>Health</td>

    <td class=ttable_head>Team</td>

	
	
	</tr>



	<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t50013')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict50013" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.The.Thrill.and.The.Agony.1080p.WEB.H264-SHREDDIE" href="torrents-details.php?id=50013&amp;hit=1"><b>UFC.245.The.Thrill.and.The.Agony.1080p.WEB.H264-SHREDDIE</b></a><div id="kt50013" style="display: none;"><BR><B>Size</B>: 836.77 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2020-01-24 20:35:13<BR><B>Completed</B>: 6<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=50013&name=UFC.245.The.Thrill.and.The.Agony.1080p.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=50013><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=50013>1</a></td>
<td class=ttable_col2 align=center>836.77 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>2</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t50012')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict50012" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.The.Thrill.and.The.Agony.720p.WEB.H264-SHREDDIE" href="torrents-details.php?id=50012&amp;hit=1"><b>UFC.245.The.Thrill.and.The.Agony.720p.WEB.H264-SHREDDIE</b></a><div id="kt50012" style="display: none;"><BR><B>Size</B>: 373.54 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2020-01-24 20:32:13<BR><B>Completed</B>: 9<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=50012&name=UFC.245.The.Thrill.and.The.Agony.720p.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=50012><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=50012>2</a></td>
<td class=ttable_col2 align=center>373.54 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>3</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t50011')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict50011" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.The.Thrill.and.The.Agony.WEB.H264-SHREDDIE" href="torrents-details.php?id=50011&amp;hit=1"><b>UFC.245.The.Thrill.and.The.Agony.WEB.H264-SHREDDIE</b></a><div id="kt50011" style="display: none;"><BR><B>Size</B>: 122.77 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2020-01-24 20:29:21<BR><B>Completed</B>: 2<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=50011&name=UFC.245.The.Thrill.and.The.Agony.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=50011><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=50011>0</a></td>
<td class=ttable_col2 align=center>122.77 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>1</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49616')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49616" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.1080p.WEB-DL.H264.Fight-BB" href="torrents-details.php?id=49616&amp;hit=1"><b>UFC.245.1080p.WEB-DL.H264.Fight-BB</b></a><div id="kt49616" style="display: none;"><BR><B>Size</B>: 8.35 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-19 09:22:00<BR><B>Completed</B>: 20<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49616&name=26599-UFC.245.1080p.WEB-DL.H264.Fight-BB.mp4.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49616><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49616>2</a></td>
<td class=ttable_col2 align=center>8.35 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>2</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49615')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49615" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.WEB-DL.H264.Fight-BB" href="torrents-details.php?id=49615&amp;hit=1"><b>UFC.245.WEB-DL.H264.Fight-BB</b></a><div id="kt49615" style="display: none;"><BR><B>Size</B>: 1.86 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-19 09:10:14<BR><B>Completed</B>: 22<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49615&name=26599-UFC.245.WEB-DL.H264.Fight-BB.mp4.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49615><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49615>1</a></td>
<td class=ttable_col2 align=center>1.86 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>3</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49614')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49614" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.720p.WEB-DL.H264.Fight-BB" href="torrents-details.php?id=49614&amp;hit=1"><b>UFC.245.720p.WEB-DL.H264.Fight-BB</b></a><div id="kt49614" style="display: none;"><BR><B>Size</B>: 5.87 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-19 09:02:06<BR><B>Completed</B>: 13<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49614&name=29334-UFC.245.720p.WEB-DL.H264.Fight-BB.mp4.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49614><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49614>2</a></td>
<td class=ttable_col2 align=center>5.87 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>1</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49599')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49599" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Post.Show.480p.x264-mSD" href="torrents-details.php?id=49599&amp;hit=1"><b>UFC.245.Post.Show.480p.x264-mSD</b></a><div id="kt49599" style="display: none;"><BR><B>Size</B>: 337.34 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-16 22:03:01<BR><B>Completed</B>: 4<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49599&name=UFC.245.Post-Show.480p.x264-mSD%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49599><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49599>0</a></td>
<td class=ttable_col2 align=center>337.34 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49598')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49598" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Post-Show.720p.WEB.H264-LEViTATE" href="torrents-details.php?id=49598&amp;hit=1"><b>UFC.245.Post-Show.720p.WEB.H264-LEViTATE</b></a><div id="kt49598" style="display: none;"><BR><B>Size</B>: 2.78 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-16 20:45:53<BR><B>Completed</B>: 3<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49598&name=UFC.245.Post-Show.720p.WEB.H264-LEViTATE%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49598><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49598>0</a></td>
<td class=ttable_col2 align=center>2.78 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49597')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49597" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Post-Show.WEB.H264-LEViTATE" href="torrents-details.php?id=49597&amp;hit=1"><b>UFC.245.Post-Show.WEB.H264-LEViTATE</b></a><div id="kt49597" style="display: none;"><BR><B>Size</B>: 1.03 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-16 20:45:28<BR><B>Completed</B>: 2<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49597&name=UFC.245.Post-Show.WEB.H264-LEViTATE%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49597><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49597>0</a></td>
<td class=ttable_col2 align=center>1.03 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49587')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49587" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Live.480p.x264-mSD" href="torrents-details.php?id=49587&amp;hit=1"><b>UFC.245.Live.480p.x264-mSD</b></a><div id="kt49587" style="display: none;"><BR><B>Size</B>: 160.12 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-16 16:52:31<BR><B>Completed</B>: 0<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49587&name=UFC.245.Live.480p.x264-mSD%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49587><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49587>0</a></td>
<td class=ttable_col2 align=center>160.12 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49586')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49586" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Live.HDTV.x264-ACES" href="torrents-details.php?id=49586&amp;hit=1"><img src='img/freeleech_star.gif' border='none' title='Freeleech' alt='[Freeleech]'> <b>UFC.245.Live.HDTV.x264-ACES</b> </font></a><div id="kt49586" style="display: none;"><BR><B>Size</B>: 157.67 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-16 16:47:00<BR><B>Completed</B>: 0<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49586&name=UFC.245.Live.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49586><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49586>0</a></td>
<td class=ttable_col2 align=center>157.67 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49585')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49585" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Live.1080p.HDTV.x264-ACES" href="torrents-details.php?id=49585&amp;hit=1"><img src='img/freeleech_star.gif' border='none' title='Freeleech' alt='[Freeleech]'> <b>UFC.245.Live.1080p.HDTV.x264-ACES</b> </font></a><div id="kt49585" style="display: none;"><BR><B>Size</B>: 726.98 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-16 16:46:11<BR><B>Completed</B>: 11<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49585&name=UFC.245.Live.1080p.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49585><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49585>2</a></td>
<td class=ttable_col2 align=center>726.98 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>1</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49584')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49584" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Live.720p.HDTV.x264-ACES" href="torrents-details.php?id=49584&amp;hit=1"><img src='img/freeleech_star.gif' border='none' title='Freeleech' alt='[Freeleech]'> <b>UFC.245.Live.720p.HDTV.x264-ACES</b> </font></a><div id="kt49584" style="display: none;"><BR><B>Size</B>: 382.94 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-16 16:46:07<BR><B>Completed</B>: 0<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49584&name=UFC.245.Live.720p.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49584><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49584>1</a></td>
<td class=ttable_col2 align=center>382.94 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49582')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49582" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Preview.480p.x264-mSD" href="torrents-details.php?id=49582&amp;hit=1"><b>UFC.245.Preview.480p.x264-mSD</b></a><div id="kt49582" style="display: none;"><BR><B>Size</B>: 80.69 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 22:26:57<BR><B>Completed</B>: 3<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49582&name=UFC.245.Preview.480p.x264-mSD%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49582><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49582>0</a></td>
<td class=ttable_col2 align=center>80.69 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49581')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49581" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Preview.1080p.HDTV.x264-ACES" href="torrents-details.php?id=49581&amp;hit=1"><b>UFC.245.Preview.1080p.HDTV.x264-ACES</b></a><div id="kt49581" style="display: none;"><BR><B>Size</B>: 409.09 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 22:21:47<BR><B>Completed</B>: 3<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49581&name=UFC.245.Preview.1080p.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49581><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49581>0</a></td>
<td class=ttable_col2 align=center>409.09 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49580')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49580" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Preview.720p.HDTV.x264-ACES" href="torrents-details.php?id=49580&amp;hit=1"><b>UFC.245.Preview.720p.HDTV.x264-ACES</b></a><div id="kt49580" style="display: none;"><BR><B>Size</B>: 197.77 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 22:21:30<BR><B>Completed</B>: 4<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49580&name=UFC.245.Preview.720p.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49580><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49580>0</a></td>
<td class=ttable_col2 align=center>197.77 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49579')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49579" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Preview.HDTV.x264-ACES" href="torrents-details.php?id=49579&amp;hit=1"><b>UFC.245.Preview.HDTV.x264-ACES</b></a><div id="kt49579" style="display: none;"><BR><B>Size</B>: 73.48 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 22:21:14<BR><B>Completed</B>: 0<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49579&name=UFC.245.Preview.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49579><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49579>0</a></td>
<td class=ttable_col2 align=center>73.48 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49576')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49576" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.PPV.720p.HDTV.x264-VERUM" href="torrents-details.php?id=49576&amp;hit=1"><b>UFC.245.PPV.720p.HDTV.x264-VERUM</b></a><div id="kt49576" style="display: none;"><BR><B>Size</B>: 5.35 GB<BR><B>Speed:</B> 56.66 kB/s<BR><B>Added:</B> 2019-12-15 13:41:15<BR><B>Completed</B>: 73<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49576&name=UFC.245.PPV.720p.HDTV.x264-VERUM.%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49576><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49576>5</a></td>
<td class=ttable_col2 align=center>5.35 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>8</b></td>
<td class=ttable_col1 align=center><font color=red><B>1</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_9.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49575')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49575" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.PPV.HDTV.x264-VERUM" href="torrents-details.php?id=49575&amp;hit=1"><b>UFC.245.PPV.HDTV.x264-VERUM</b></a><div id="kt49575" style="display: none;"><BR><B>Size</B>: 2.40 GB<BR><B>Speed:</B> 38.68 kB/s<BR><B>Added:</B> 2019-12-15 12:11:22<BR><B>Completed</B>: 111<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49575&name=UFC.245.PPV.HDTV.x264-VERUM.%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49575><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49575>8</a></td>
<td class=ttable_col2 align=center>2.40 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>5</b></td>
<td class=ttable_col1 align=center><font color=red><B>1</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_8.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49574')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49574" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.720p.WEB.H264.Fight-BB" href="torrents-details.php?id=49574&amp;hit=1"><b>UFC.245.720p.WEB.H264.Fight-BB</b></a><div id="kt49574" style="display: none;"><BR><B>Size</B>: 4.66 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 11:22:31<BR><B>Completed</B>: 243<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49574&name=29334-UFC.245.720p.WEB.H264.Fight-BB.mkv.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49574><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49574>24</a></td>
<td class=ttable_col2 align=center>4.66 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>11</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49573')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49573" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Prelims.1080p.WEB.h264-VERUM" href="torrents-details.php?id=49573&amp;hit=1"><b>UFC.245.Prelims.1080p.WEB.h264-VERUM</b></a><div id="kt49573" style="display: none;"><BR><B>Size</B>: 4.65 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 11:09:41<BR><B>Completed</B>: 159<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49573&name=UFC.245.Prelims.1080p.WEB.h264-VERUM%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49573><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49573>2</a></td>
<td class=ttable_col2 align=center>4.65 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>2</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49572')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49572" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.PPV.Usman.vs.Covington.HDTV.x264-Star" href="torrents-details.php?id=49572&amp;hit=1"><b>UFC.245.PPV.Usman.vs.Covington.HDTV.x264-Star</b></a><div id="kt49572" style="display: none;"><BR><B>Size</B>: 3.03 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 10:18:15<BR><B>Completed</B>: 149<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49572&name=UFC.245.PPV.Usman.vs.Covington.HDTV.x264-Star.mp4.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49572><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49572>4</a></td>
<td class=ttable_col2 align=center>3.03 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>3</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#15'><img src="https://mma-torrents.com/image.php?id=8kmt0Cjx8GU1p.png" alt="" title="Kung Fu Legends" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49569')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49569" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.540p.WEB.H264.Fight-BB" href="torrents-details.php?id=49569&amp;hit=1"><b>UFC.245.540p.WEB.H264.Fight-BB</b></a><div id="kt49569" style="display: none;"><BR><B>Size</B>: 3.06 GB<BR><B>Speed:</B> 150.12 kB/s<BR><B>Added:</B> 2019-12-15 09:30:12<BR><B>Completed</B>: 338<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49569&name=26599-UFC.245.540p.WEB.H264.Fight-BB.mkv.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49569><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49569>23</a></td>
<td class=ttable_col2 align=center>3.06 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>16</b></td>
<td class=ttable_col1 align=center><font color=red><B>2</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_9.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49566')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49566" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.PPV.1080p.WEB.h264-VERUM" href="torrents-details.php?id=49566&amp;hit=1"><b>UFC.245.PPV.1080p.WEB.h264-VERUM</b></a><div id="kt49566" style="display: none;"><BR><B>Size</B>: 10.54 GB<BR><B>Speed:</B> 721.52 kB/s<BR><B>Added:</B> 2019-12-15 07:51:20<BR><B>Completed</B>: 473<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49566&name=UFC.245.PPV.1080p.WEB.h264-VERUM%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49566><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49566>35</a></td>
<td class=ttable_col2 align=center>10.54 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>24</b></td>
<td class=ttable_col1 align=center><font color=red><B>2</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_9.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49565')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49565" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Early.Prelims.480p.x264-mSD" href="torrents-details.php?id=49565&amp;hit=1"><b>UFC.245.Early.Prelims.480p.x264-mSD</b></a><div id="kt49565" style="display: none;"><BR><B>Size</B>: 925.77 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 06:32:30<BR><B>Completed</B>: 65<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49565&name=UFC.245.Early.Prelims.480p.x264-mSD%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49565><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49565>6</a></td>
<td class=ttable_col2 align=center>925.77 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>5</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49554')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49554" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Early.Prelims.720p.WEB.H264-LEViTATE" href="torrents-details.php?id=49554&amp;hit=1"><b>UFC.245.Early.Prelims.720p.WEB.H264-LEViTATE</b></a><div id="kt49554" style="display: none;"><BR><B>Size</B>: 5.11 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 06:16:51<BR><B>Completed</B>: 16<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49554&name=UFC.245.Early.Prelims.720p.WEB.H264-LEViTATE.%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49554><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49554>2</a></td>
<td class=ttable_col2 align=center>5.11 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>2</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49553')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49553" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Early.Prelims.WEB.H264-LEViTATE" href="torrents-details.php?id=49553&amp;hit=1"><b>UFC.245.Early.Prelims.WEB.H264-LEViTATE</b></a><div id="kt49553" style="display: none;"><BR><B>Size</B>: 1.84 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 06:09:57<BR><B>Completed</B>: 22<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49553&name=UFC.245.Early.Prelims.WEB.H264-LEViTATE%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49553><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49553>1</a></td>
<td class=ttable_col2 align=center>1.84 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49552')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49552" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Prelims.720p.HDTV.x264-WH" href="torrents-details.php?id=49552&amp;hit=1"><b>UFC.245.Prelims.720p.HDTV.x264-WH</b></a><div id="kt49552" style="display: none;"><BR><B>Size</B>: 3.01 GB<BR><B>Speed:</B> 58.19 kB/s<BR><B>Added:</B> 2019-12-15 03:46:09<BR><B>Completed</B>: 134<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49552&name=UFC.245.Prelims.720p.HDTV.x264-WH.%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49552><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49552>6</a></td>
<td class=ttable_col2 align=center>3.01 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>7</b></td>
<td class=ttable_col1 align=center><font color=red><B>1</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_9.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49550')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49550" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Prelims.1080p.WEB-DL.H264.Fight-BB" href="torrents-details.php?id=49550&amp;hit=1"><b>UFC.245.Prelims.1080p.WEB-DL.H264.Fight-BB</b></a><div id="kt49550" style="display: none;"><BR><B>Size</B>: 4.99 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 03:34:55<BR><B>Completed</B>: 85<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49550&name=26599-UFC.245.Prelims.1080p.WEB-DL.H264.Fight-BB.mp4.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49550><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49550>10</a></td>
<td class=ttable_col2 align=center>4.99 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>4</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49549')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49549" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Prelims.WEB.H264-SHREDDIE" href="torrents-details.php?id=49549&amp;hit=1"><b>UFC.245.Prelims.WEB.H264-SHREDDIE</b></a><div id="kt49549" style="display: none;"><BR><B>Size</B>: 1.44 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 03:30:25<BR><B>Completed</B>: 127<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49549&name=UFC.245.Prelims.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49549><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49549>7</a></td>
<td class=ttable_col2 align=center>1.44 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>5</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49548')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49548" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Prelims.WEB-DL.H264.Fight-BB" href="torrents-details.php?id=49548&amp;hit=1"><b>UFC.245.Prelims.WEB-DL.H264.Fight-BB</b></a><div id="kt49548" style="display: none;"><BR><B>Size</B>: 1.76 GB<BR><B>Speed:</B> 38.05 kB/s<BR><B>Added:</B> 2019-12-15 03:28:16<BR><B>Completed</B>: 150<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49548&name=26599-UFC.245.Prelims.WEB-DL.H264.Fight-BB.mp4.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49548><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49548>9</a></td>
<td class=ttable_col2 align=center>1.76 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>7</b></td>
<td class=ttable_col1 align=center><font color=red><B>1</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_9.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49547')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49547" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Prelims.Repack.1080p.WEB.H264-SHREDDIE" href="torrents-details.php?id=49547&amp;hit=1"><b>UFC.245.Prelims.Repack.1080p.WEB.H264-SHREDDIE</b></a><div id="kt49547" style="display: none;"><BR><B>Size</B>: 5.14 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 03:24:58<BR><B>Completed</B>: 16<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49547&name=UFC.245.Prelims.Repack.1080p.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49547><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49547>2</a></td>
<td class=ttable_col2 align=center>5.14 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>2</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49546')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49546" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Prelims.720p.WEB-DL.H264.Fight-BB" href="torrents-details.php?id=49546&amp;hit=1"><b>UFC.245.Prelims.720p.WEB-DL.H264.Fight-BB</b></a><div id="kt49546" style="display: none;"><BR><B>Size</B>: 3.12 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 03:21:24<BR><B>Completed</B>: 115<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49546&name=29334-UFC.245.Prelims.720p.WEB-DL.H264.Fight-BB.mp4.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49546><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49546>9</a></td>
<td class=ttable_col2 align=center>3.12 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>5</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49545')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49545" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Prelims.Repack.720p.WEB.H264-SHREDDIE" href="torrents-details.php?id=49545&amp;hit=1"><b>UFC.245.Prelims.Repack.720p.WEB.H264-SHREDDIE</b></a><div id="kt49545" style="display: none;"><BR><B>Size</B>: 3.97 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 03:20:05<BR><B>Completed</B>: 21<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49545&name=UFC.245.Prelims.Repack.720p.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49545><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49545>0</a></td>
<td class=ttable_col2 align=center>3.97 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49544')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49544" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Early.Prelims.WEB.H264-SHREDDIE" href="torrents-details.php?id=49544&amp;hit=1"><b>UFC.245.Early.Prelims.WEB.H264-SHREDDIE</b></a><div id="kt49544" style="display: none;"><BR><B>Size</B>: 1.19 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 01:45:26<BR><B>Completed</B>: 47<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49544&name=UFC.245.Early.Prelims.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49544><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49544>4</a></td>
<td class=ttable_col2 align=center>1.19 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>2</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49542')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49542" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Early.Prelims.WEB-DL.H264.Fight-BB" href="torrents-details.php?id=49542&amp;hit=1"><b>UFC.245.Early.Prelims.WEB-DL.H264.Fight-BB</b></a><div id="kt49542" style="display: none;"><BR><B>Size</B>: 1.21 GB<BR><B>Speed:</B> 18.93 kB/s<BR><B>Added:</B> 2019-12-15 01:36:50<BR><B>Completed</B>: 108<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49542&name=26599-UFC.245.Early.Prelims.WEB-DL.H264.Fight-BB.mp4.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49542><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49542>5</a></td>
<td class=ttable_col2 align=center>1.21 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>5</b></td>
<td class=ttable_col1 align=center><font color=red><B>1</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_8.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49540')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49540" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Early.Prelims.720p.WEB-DL.H264.Fight-BB" href="torrents-details.php?id=49540&amp;hit=1"><b>UFC.245.Early.Prelims.720p.WEB-DL.H264.Fight-BB</b></a><div id="kt49540" style="display: none;"><BR><B>Size</B>: 2.10 GB<BR><B>Speed:</B> 34.52 kB/s<BR><B>Added:</B> 2019-12-15 01:32:23<BR><B>Completed</B>: 114<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49540&name=29334-UFC.245.Early.Prelims.720p.WEB-DL.H264.Fight-BB.mp4.torrent%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49540><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49540>8</a></td>
<td class=ttable_col2 align=center>2.10 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>6</b></td>
<td class=ttable_col1 align=center><font color=red><B>1</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_9.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49539')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49539" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Early.Prelims.720p.WEB.H264-SHREDDIE" href="torrents-details.php?id=49539&amp;hit=1"><b>UFC.245.Early.Prelims.720p.WEB.H264-SHREDDIE</b></a><div id="kt49539" style="display: none;"><BR><B>Size</B>: 3.11 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 01:29:53<BR><B>Completed</B>: 7<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49539&name=UFC.245.Prelims.720p.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49539><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49539>0</a></td>
<td class=ttable_col2 align=center>3.11 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>1</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49538')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49538" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Early.Prelims.1080p.WEB.H264-SHREDDIE" href="torrents-details.php?id=49538&amp;hit=1"><b>UFC.245.Early.Prelims.1080p.WEB.H264-SHREDDIE</b></a><div id="kt49538" style="display: none;"><BR><B>Size</B>: 3.38 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-15 01:22:48<BR><B>Completed</B>: 33<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49538&name=UFC.245.Prelims.1080p.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49538><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49538>5</a></td>
<td class=ttable_col2 align=center>3.38 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>2</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49525')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49525" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Embedded-Vlog.Series-Episode.5.720p.WEBRip.h264-TJ" href="torrents-details.php?id=49525&amp;hit=1"><img src='img/freeleech_star.gif' border='none' title='Freeleech' alt='[Freeleech]'> <b>UFC.245.Embedded-Vlog.Series-Episode.5.720p.WEBRip.h264-TJ</b> </font></a><div id="kt49525" style="display: none;"><BR><B>Size</B>: 205.71 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-14 08:16:24<BR><B>Completed</B>: 6<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49525&name=UFC.245.Embedded-Vlog.Series-Episode.5.720p.WEBRip.h264-TJ.mp4.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49525><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49525>3</a></td>
<td class=ttable_col2 align=center>205.71 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#15'><img src="https://mma-torrents.com/image.php?id=8kmt0Cjx8GU1p.png" alt="" title="Kung Fu Legends" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49522')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49522" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Weigh-In.1080p.WEB.H264-SHREDDIE" href="torrents-details.php?id=49522&amp;hit=1"><b>UFC.245.Weigh-In.1080p.WEB.H264-SHREDDIE</b></a><div id="kt49522" style="display: none;"><BR><B>Size</B>: 1.17 GB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-14 02:15:37<BR><B>Completed</B>: 6<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49522&name=UFC.245.Weigh-In.1080p.WEB.H264-SHREDDIE.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49522><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49522>3</a></td>
<td class=ttable_col2 align=center>1.17 GB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#14'><img src="/img/teams/elite_team.jpg" alt="" title="Elite Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49520')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49520" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Fight.Camp.480p.x264-mSD" href="torrents-details.php?id=49520&amp;hit=1"><b>UFC.245.Fight.Camp.480p.x264-mSD</b></a><div id="kt49520" style="display: none;"><BR><B>Size</B>: 153.63 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-14 01:24:36<BR><B>Completed</B>: 1<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49520&name=UFC.245.Fight.Camp.480p.x264-mSD%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49520><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49520>0</a></td>
<td class=ttable_col2 align=center>153.63 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49519')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49519" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Fight.Camp.720p.HDTV.x264-ACES" href="torrents-details.php?id=49519&amp;hit=1"><b>UFC.245.Fight.Camp.720p.HDTV.x264-ACES</b></a><div id="kt49519" style="display: none;"><BR><B>Size</B>: 380.56 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-14 01:15:37<BR><B>Completed</B>: 10<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49519&name=UFC.245.Fight.Camp.720p.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49519><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49519>0</a></td>
<td class=ttable_col2 align=center>380.56 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49518')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49518" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Fight.Camp.HDTV.x264-ACES" href="torrents-details.php?id=49518&amp;hit=1"><b>UFC.245.Fight.Camp.HDTV.x264-ACES</b></a><div id="kt49518" style="display: none;"><BR><B>Size</B>: 151.11 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-14 01:15:33<BR><B>Completed</B>: 3<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49518&name=UFC.245.Fight.Camp.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49518><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49518>0</a></td>
<td class=ttable_col2 align=center>151.11 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49517')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49517" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Fight.Camp.1080p.HDTV.x264-ACES" href="torrents-details.php?id=49517&amp;hit=1"><b>UFC.245.Fight.Camp.1080p.HDTV.x264-ACES</b></a><div id="kt49517" style="display: none;"><BR><B>Size</B>: 734.68 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-14 01:15:11<BR><B>Completed</B>: 2<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49517&name=UFC.245.Fight.Camp.1080p.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49517><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49517>1</a></td>
<td class=ttable_col2 align=center>734.68 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49506')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49506" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Inside.The.Octagon.1080p.HDTV.x264-ACES" href="torrents-details.php?id=49506&amp;hit=1"><img src='img/freeleech_star.gif' border='none' title='Freeleech' alt='[Freeleech]'> <b>UFC.245.Inside.The.Octagon.1080p.HDTV.x264-ACES</b> </font></a><div id="kt49506" style="display: none;"><BR><B>Size</B>: 701.42 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-13 01:38:08<BR><B>Completed</B>: 5<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49506&name=UFC.245.Inside.The.Octagon.1080p.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49506><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49506>2</a></td>
<td class=ttable_col2 align=center>701.42 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>1</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_10.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49505')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49505" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Inside.The.Octagon.480p.x264-mSD" href="torrents-details.php?id=49505&amp;hit=1"><img src='img/freeleech_star.gif' border='none' title='Freeleech' alt='[Freeleech]'> <b>UFC.245.Inside.The.Octagon.480p.x264-mSD</b> </font></a><div id="kt49505" style="display: none;"><BR><B>Size</B>: 133.55 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-13 01:36:56<BR><B>Completed</B>: 0<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49505&name=UFC.245.Inside.The.Octagon.480p.x264-mSD%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49505><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49505>0</a></td>
<td class=ttable_col2 align=center>133.55 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49504')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49504" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Inside.The.Octagon.HDTV.x264-ACES" href="torrents-details.php?id=49504&amp;hit=1"><img src='img/freeleech_star.gif' border='none' title='Freeleech' alt='[Freeleech]'> <b>UFC.245.Inside.The.Octagon.HDTV.x264-ACES</b> </font></a><div id="kt49504" style="display: none;"><BR><B>Size</B>: 133.87 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-13 01:31:20<BR><B>Completed</B>: 0<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49504&name=UFC.245.Inside.The.Octagon.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49504><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49504>0</a></td>
<td class=ttable_col2 align=center>133.87 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=96"><img border="0"src="https://mma-torrents.com/images/categories/ufchd.png" alt="UFC/HD" title="Events - UFC/HD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49503')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49503" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Inside.The.Octagon.720p.HDTV.x264-ACES" href="torrents-details.php?id=49503&amp;hit=1"><img src='img/freeleech_star.gif' border='none' title='Freeleech' alt='[Freeleech]'> <b>UFC.245.Inside.The.Octagon.720p.HDTV.x264-ACES</b> </font></a><div id="kt49503" style="display: none;"><BR><B>Size</B>: 356.51 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-13 01:30:24<BR><B>Completed</B>: 6<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49503&name=UFC.245.Inside.The.Octagon.720p.HDTV.x264-ACES%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49503><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49503>1</a></td>
<td class=ttable_col2 align=center>356.51 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
<tr>
<td class=ttable_col1 align=center valign=middle><a href="torrents.php?cat=103"><img border="0"src="https://mma-torrents.com/images/categories/ufc2011.png" alt="UFC/SD" title="Events - UFC/SD"/></a></td>
<td class=ttable_col2 nowrap><a href="javascript: klappe_torrent('t49502')"><img border="0" src="https://mma-torrents.com/images/plus.gif" id="pict49502" alt="Show/Hide" class="showthecross">&nbsp;<a title="UFC.245.Countdown.480p.x264-mSD" href="torrents-details.php?id=49502&amp;hit=1"><b>UFC.245.Countdown.480p.x264-mSD</b></a><div id="kt49502" style="display: none;"><BR><B>Size</B>: 371.18 MB<BR><B>Speed:</B> No Activity<BR><B>Added:</B> 2019-12-12 23:00:05<BR><B>Completed</B>: 9<BR></div></td>
<td class=ttable_col1 align=center><a href="download.php?id=49502&name=UFC.245.Countdown.480p.x264-mSD%5Bmma%5D.torrent"><img src=https://mma-torrents.com/images/icon_download.gif border=0 alt="Download .torrent"></a></td><td class=ttable_col1 align=center><a href=nfo-view.php?id=49502><img  src=https://mma-torrents.com/images/icon_nfo.gif border=0 alt='View NFO'></a></td><td class=ttable_col1 align=center><font size=1 face=Verdana><a href=comments.php?type=torrent&id=49502>2</a></td>
<td class=ttable_col2 align=center>371.18 MB</td>
<td class=ttable_col2 align=center><b><font color=green><B>0</b></td>
<td class=ttable_col1 align=center><font color=red><B>0</b></td>
<td class=ttable_col1 align=center><img src=https://mma-torrents.com/images/health_1.gif></td>
<td class=ttable_col2 align=center><a href='teams-view.php#13'><img src="/img/teams/bjjteam.jpg" alt="" title="The Brazilian Jiu-Jitsu Team" height="30" width="30" border="0" /></a></td>
</tr>
</table><BR>
<p align="center"><span class=newlink>1&nbsp;-&nbsp;50</span> | <a href="torrents.php?incldead=1&amp;search=ufc+245&amp;sort=id&order=desc&page=1"><span class=newlink>51&nbsp;-&nbsp;59</span></a><br /><br><span class=newlink>&lt;&lt;&nbsp;Prev</span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="torrents.php?incldead=1&amp;search=ufc+245&amp;sort=id&order=desc&page=1"><span class=newlink>Next&nbsp;&gt;&gt;</span><br></a></p>
</div>
</td>
<td width='4' valign='top' background='themes/NB-MMA/images/fmr.gif'><img src='themes/NB-MMA/images/fmrt.gif' width='4' height='12' /></td>
</tr>
</table>
</div>
<div>
<div class='fram-bl'></div>
<div class='fram-br'></div>
<div class='fram-bm'></div>
</div>
</div>	</TD><!-- END MAIN CONTENT AREA -->

	    <!-- RIGHT COLUMN -->
    <TD id="rcol" vAlign="top" width="180">
    <div class='blok'>
<div class='blok-t'>MMA Events</div>
<div class='blok-m'>
<div class='blok-content'><font size="0"><table cellpadding="0" cellspacing="0" border="0" width="100%" align="center"><tr><td align=center style="line-height:0.7;"><a href="/torrents.php?search=LFA+83" title="Search for LFA+83 All releases"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><a href="/torrents.php?search=LFA+83+720" title="Search for LFA+83+720"><img src="/images/all3_search.png" width="50" height="3" border="0" style="border-radius: 50%;" /></a><a href="/torrents.php?search=LFA+83+1080" title="Search for LFA+83+1080"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><br><a href="/foru/viewtopic.php?f=13&t=9389" target="_blank"><img src="/images/events_small/lfa83sml.gif" alt="LFA 83 - Jackson vs. Chaulet - Mar 6" title="LFA 83 - Jackson vs. Chaulet - Mar 6" width="150" height="75" border="0" /></a></td></tr><tr><td align=center style="line-height:0.7;"><a href="/torrents.php?search=Invicta+FC" title="Search for Invicta+FC All releases"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><a href="/torrents.php?search=Invicta+FC+720" title="Search for Invicta+FC+720"><img src="/images/all3_search.png" width="50" height="3" border="0" style="border-radius: 50%;" /></a><a href="/torrents.php?search=Invicta+FC+1080" title="Search for Invicta+FC+1080"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><br><a href="/foru/viewtopic.php?f=13&t=9388" target="_blank"><img src="/images/events_small/invictaphoenix3sml2.gif" alt="Invicta FC - Phoenix Series 3 - Mar 6" title="Invicta FC - Phoenix Series 3 - Mar 6" width="150" height="75" border="0" /></a></td></tr><tr><td align=center style="line-height:0.7;"><a href="/torrents.php?search=UFC+248" title="Search for UFC+248 All releases"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><a href="/torrents.php?search=UFC+248+720" title="Search for UFC+248+720"><img src="/images/all3_search.png" width="50" height="3" border="0" style="border-radius: 50%;" /></a><a href="/torrents.php?search=UFC+248+1080" title="Search for UFC+248+1080"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><br><a href="/foru/viewtopic.php?f=13&t=9393" target="_blank"><img src="/images/events_small/ufc248sml.gif" alt="UFC 248 - Adesanya vs. Romero - Mar 7" title="UFC 248 - Adesanya vs. Romero - Mar 7" width="150" height="75" border="0" /></a></td></tr><tr><td align=center style="line-height:0.7;"><a href="/torrents.php?search=Cage+Warriors+112" title="Search for Cage+Warriors+112 All releases"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><a href="/torrents.php?search=Cage+Warriors+112+720" title="Search for Cage+Warriors+112+720"><img src="/images/all3_search.png" width="50" height="3" border="0" style="border-radius: 50%;" /></a><a href="/torrents.php?search=Cage+Warriors+112+1080" title="Search for Cage+Warriors+112+1080"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><br><a href="/foru/viewtopic.php?f=13&t=9392" target="_blank"><img src="/images/events_small/cw112sml.gif" alt="Cage Warriors 112 - Cartwright vs. Bilic - Mar 7" title="Cage Warriors 112 - Cartwright vs. Bilic - Mar 7" width="150" height="75" border="0" /></a></td></tr><tr><td align=center style="line-height:0.7;"><a href="/torrents.php?search=Eternal+MMA+52" title="Search for Eternal+MMA+52 All releases"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><a href="/torrents.php?search=Eternal+MMA+52+720" title="Search for Eternal+MMA+52+720"><img src="/images/all3_search.png" width="50" height="3" border="0" style="border-radius: 50%;" /></a><a href="/torrents.php?search=Eternal+MMA+52+1080" title="Search for Eternal+MMA+52+1080"><img src="/images/all_search.png" width="50" height="3" border="0" style="border-radius: 50%;"/></a><br><a href="/foru/viewtopic.php?f=13&t=9390" target="_blank"><img src="/images/events_small/eternalmma52sml.gif" alt="Eternal MMA 52 - Ross vs. Erceg - Mar 6" title="Eternal MMA 52 - Ross vs. Erceg - Mar 6" width="150" height="75" border="0" /></a></td></tr></table></font></div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>MMACal</div>
<div class='blok-m'>
<div class='blok-content'><center>
<a href="cal.php" title="MMA Events Calendar"><img width="79" height="79" src="images/calendar-icon.png"/></a>
</center>
</div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>MMATV</div>
<div class='blok-m'>
<div class='blok-content'><center>
<a href="TV.php" title="All the MMA channels in one place!"><img width="64" height="64" src="images/tv-screen.png"/></a>
</center>
</div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>MMATube</div>
<div class='blok-m'>
<div class='blok-content'><center>
<a href="yt.php" title="Watch and add videos on the MMATube!"><img src="images/play-icon.png"/></a>
</center>
</div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>Poll</div>
<div class='blok-m'>
<div class='blok-content'><p align=center><b>UFC Fight Night 169 - Benavidez vs. Figueiredo was</b></p>
<form method=post action='?search=ufc+245&incldead=1'>
<input type=hidden name=act value='takepoll'><input type=radio name=choice value=0>Awesome<br>
<input type=radio name=choice value=1>Great<br>
<input type=radio name=choice value=2>Good<br>
<input type=radio name=choice value=3>Ok<br>
<input type=radio name=choice value=4>Bad<br>
<br><p align=center><input type=submit value='Vote!' class=btn></p></form></div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>Donate</div>
<div class='blok-m'>
<div class='blok-content'>
<BR><BR><CENTER>
<a href="/donate-entry.php"><img src="/img/15star1c.gif" /></a><br><br>
<a href="/donate-entry.php">Become VIP!</a><br>
<a href="/donate-entry.php">Boost your ratio</a><br>
<a href="/donate-entry.php">Support the tracker</a><br>
<a href="/donate-entry.php">Earn Free GB's</a><br>
<a href="/donate-entry.php">Invite friends</a><br><br>
<a href="/donate-entry.php"><img src="/img/cards_accepted.png" border="none" alt="DONATE NOW!" title="Click here for donation options" width="100" height="18" /></a>
<p>March: </p>
<img height=20 src='images/health_2.gif' alt='18%'><br/>Donations: (18 %)
</CENTER><BR>

</div>
<div class='blok-bottom'></div>
</div>
</div>




<div class='blok'>
<div class='blok-t'>MMA Comps</div>
<div class='blok-m'>
<div class='blok-content'><center><a href="competitions.php" title="Competitions"><img src="images/competitions.png"></a><br><font size="0"><table cellpadding="0" cellspacing="0" border="0" width="100%" align="center"><a href="competition.php?cid=90" target="_blank"><img src="https://mma-torrents.com/imagehost/6e6258e3d164aa2fec7df7aeb4b9ad6c.jpg" alt="UFC 248 - Adesanya vs. Romero" title="UFC 248 - Adesanya vs. Romero Competition" width="150" height="75" border="0" /></a><a href="competition.php?cid=89" target="_blank"><img src="https://mma-torrents.com/imagehost/ce5fcaa3db6ee24e38e1d74583852300.jpg" alt="UFC Fight Night 169 - Benavidez vs. Figueiredo" title="UFC Fight Night 169 - Benavidez vs. Figueiredo Competition" width="150" height="75" border="0" /></a><a href="competition.php?cid=88" target="_blank"><img src="https://mma-torrents.com/imagehost/aed3060afe3270129d11e2f7398d22d2.jpg" alt="UFC Fight Night 167 - Anderson vs. Blachowicz 2" title="UFC Fight Night 167 - Anderson vs. Blachowicz 2 Competition" width="150" height="75" border="0" /></a></center></table></font></div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>Stats</div>
<div class='blok-m'>
<div class='blok-content'><div align=left><br><B>Torrents</B><br><small>Tracking:<B> 44,785 Torrents</b></small><br><small>New Today:<B> 49</b></small><br><small>Seeders:<B> 29,023</b></small><br><small>Leechers:<B> 1,807</b></small><br><br><B>Members</B><br><small>Visited Today: <B>1,314</b></small><br><br><B>Online</B><br><small>Total:<B> 211</b></small><br><small>Members:<B> 115</b></small><br><small>Guests:<B> 96</b></small><br><br></div></div>
<div class='blok-bottom'></div>
</div>
</div><div class='blok'>
<div class='blok-t'>Seedbox</div>
<div class='blok-m'>
<div class='blok-content'><center>
<a title="Use promotional code mmatorrents to get a -10% discount on seedbox" href="http://www.dentoo.info/shop/" target="_blank">
<img border="none" src="https://mma-torrents.com/image.php?id=Yy5N8bry74yS4.png" /></a>
</center>
</div>
<div class='blok-bottom'></div>
</div>
</div>
    </TD>
    <!-- END RIGHT COLUMN -->
    
    </TR>
</TABLE>
<!-- END CONTENT CODE -->
    </td>
    <td width='4' background='themes/NB-MMA/images/main-r.png'>&nbsp;</td>
  </tr>
</table>

  <table width='100%' border='0' cellspacing='0' cellpadding='0'>
    <tr>
      <td width='16' height='75'><img src='themes/NB-MMA/images/foot-l.png' width='16' height='75' /></td>
      <td height='75' valign='top' background='themes/NB-MMA/images/foot-m.png'>
      <div id="credits">
<!-- START FOOTER CODE -->
<br>Theme By: <a href="http://nikkbu.info" target="_blank">Nikkbu</a><!-- END FOOTER CODE -->
      </div>
      <table width='135' border='0' align='right' cellpadding='5' cellspacing='0'>
  <tr>
    <td width='45' height='45'><a id='twitter' href='https://twitter.com/mmatorrentscom' target='_blank'><img src='themes/NB-MMA/images/blank.gif' alt='Twitter' border='none'></a></td>
    <td width='45' height='45'><a id='rss' href='https://mma-torrents.com/rss.php'><img src='themes/NB-MMA/images/blank.gif' alt='RSS Feed' border='none'></a></td>
        <td width='45' height='45'><a id='facebook' href='https://www.facebook.com/pg/mmatorrentsupdates/posts/' target='_blank'><img src='themes/NB-MMA/images/blank.gif' alt='Twitter' border='none'></a></td>
      </tr>
    </table>
      </td>
      <td width='16' height='75'><img src='themes/NB-MMA/images/foot-r.png' width='16' height='75' /></td>
    </tr>
  </table>
  <br /><br /><br />
  </div></td>
  </tr>
</table>
</div>
</BODY>
<script type="text/javascript"> //<![CDATA[
  var tlJsHost = ((window.location.protocol == "https:") ? "https://secure.trust-provider.com/" : "http://www.trustlogo.com/");
  document.write(unescape("%3Cscript src='" + tlJsHost + "trustlogo/javascript/trustlogo.js' type='text/javascript'%3E%3C/script%3E"));
//]]></script>
<script language="JavaScript" type="text/javascript">
  TrustLogo("https://www.positivessl.com/images/seals/positivessl_trust_seal_sm_124x32.png", "POSDV", "none");
</script>
</HTML>`

Reply:
Thank you for taking the time to review my crude attempt.